### DISCUSSION TITLE
Release date?

### DISCUSSION THREAD

**Discussion by saankim** [flowershow/flowershow#148] [General and Help] [answered]
Is there any ETA you planning to?

It's hard to just waiting for such a great app like this!

**Comment by rufuspollock** [comment]
@saankim thank-you for your enthusiasm 😄

What we have is already usable and we hope to have a public alpha if you want to dive into the repo and ask questions. We will have a publicly usable self service option in next days / weeks (by end of sep at latest). 🚀

**Comment by rufuspollock** [comment, marked as answer]
Flowershow Alpha was released 10d ago on 20th October https://github.com/flowershow/flowershow/discussions/215
